### PR TITLE
Initial support for BGP Prefix SID Type 5 (L3 service)

### PR DESCRIPTION
**Pull request by sbezverk**
This PR adds initial object definitions required for BGP Prefix SID Type 5, Layer 3 Service support and unit tests.

https://tools.ietf.org/html/draft-dawra-bess-srv6-services-02

The following PR will extend API to allow programming of a prefix with BGP Prefix SID Type 5 attribute.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,6 +8,7 @@ require (
 	github.com/eapache/channels v1.1.0
 	github.com/eapache/queue v1.0.2 // indirect
 	github.com/fsnotify/fsnotify v1.4.2 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```
Comment (fujita) [inline go.mod:11]: where this is used?

**Comment by fujita** [comment]
Thanks a lot!

Can you fix staticcheck warnings?
https://travis-ci.org/github/osrg/gobgp/jobs/684169522

File: go.mod (hunk 1 above)
Comment (sbezverk) [reply go.mod]: Sorry, left over from my debugging, cleaned up in the latest version of PR

**Comment by sbezverk** [comment]
@fujita Thank you very much for your prompt review. I have addressed all comments. PTAL.

**Comment by fujita** [comment]
Thanks a lot! Keep up the good work!

**Pull request merged**